LEAF_Request_Portal/admin/index.php

```diff
@@ -2,206 +2,167 @@
 /*
  * As a work of the United States government, this project is in the public domain within the United States.
  */
-
 /*
     Index for everything
     Date Created: September 11, 2007
-
 */
-
 error_reporting(E_ERROR);
-
 require_once '../globals.php';
 require_once LIB_PATH . '/loaders/Leaf_autoloader.php';
-
 header('X-UA-Compatible: IE=edge');
-
 $login->setBaseDir('../');
-
```

Comment:
Why did you remove every blank line in this file? Blank lines do help in readability